src/engraving/dom/systemdivider.cpp

```diff
@@ -62,11 +62,18 @@ SystemDivider::SystemDivider(const SystemDivider& sd)
 void SystemDivider::setDividerType(SystemDivider::Type v)
 {
     m_dividerType = v;
+
     if (v == SystemDivider::Type::LEFT) {
         setOffset(PointF(style().styleD(Sid::dividerLeftX), style().styleD(Sid::dividerLeftY)));
     } else {
         setOffset(PointF(style().styleD(Sid::dividerRightX), style().styleD(Sid::dividerRightY)));
     }
+
+    if (v == SystemDivider::Type::LEFT) {
+        setSym(SymNames::symIdByName(style().styleSt(Sid::dividerLeftSym)), score()->engravingFont());
```

Comment:
We should test that the SymId of existing SystemDividers is still updated correctly when the user changes the style setting. I fear it's not.

Reply:
added `SystemDivider::styleChanged`